Src/Plugins/Tests.Security/TestData/SecurePlaintextSecrets/ExpectedOutputs/SEC101_006.GitHubLegacyPat.sarif

```diff
@@ -0,0 +1,280 @@
+{
```

Comment:
Is this sarif file addition expected with this change? Was it inadvertently excluded during a previous PR?